credit_card_checker/credit_card_checker.rb

```diff
@@ -1,5 +1,101 @@
+require './bank_data'
+
+# Class provides methods for validation of credit card numbers based on luhn algorithm
+#   and specific bank information like IIN ranges and code lengthes.
+# Specific bank data is stored in bank_data.rb.
+
 class CreditCardChecker
+
+  # Method accepts string or int code numbers.
```

Comment:
implement @errors attribute for checker object that returns information why check fails